gamestonk_terminal/portfolio/degiro_api.py

```diff
@@ -0,0 +1,484 @@
+
+        # SETUP REQUEST
+        isin = ns_parser.isin
+        request = NewsByCompany.Request(
+            isin=isin,
+            limit=10,
+            offset=0,
+            languages="en,fr",
+        )
+
+        # FETCH DATA
+        news = trading_api.get_news_by_company(
+            request=request,
+            raw=True,
+        )
+
+        # DISPLAY DATA
+        for article in news["data"]["items"]:
+            print("date", article["date"])
+            print("title", article["title"])
+            print("content", article["content"])
+            print("isins", article["isins"])
+            print("---")
+
+    def pending(self, _):
+        """Display pending orders."""
+
+        # GET ATTRIBUTES
+        trading_api = self.__trading_api
+
+        # SETUP REQUEST
+        request_list = Update.RequestList()
+        request_list.values.extend(
+            [
+                Update.Request(option=Update.Option.ORDERS, last_updated=0),
+            ]
+        )
+
+        # FETCH DATA
+        update = trading_api.get_update(request_list=request_list)
+
+        # FORMAT DATA
+        update_dict = pb_handler.message_to_dict(message=update)
+        orders_df = pd.DataFrame(update_dict["orders"]["values"])
+
+        # DISPLAY DATA
+        if orders_df.shape[0] == 0:
+            print("No pending orders.")
+        else:
+            print(orders_df)
+
+    def switch(self, an_input: str):
+        """Process and dispatch input
+
+        Returns
+        -------
+        True, False or None
+            False - quit the menu
+            True - quit the program
+            None - continue in the menu
+        """
+
+        try:
+            degiro_parser = self.__degiro_parser
+
+            (known_args, other_args) = degiro_parser.parse_known_args(an_input.split())
+
+            return getattr(
+                self,
+                known_args.cmd,
+                lambda: "Command not recognized!",
+            )(other_args)
+        except Exception as e:
+            print(e)
+            print("")
+
+            return None
+
+
+def menu():
+    """Degiro Menu"""
+
+    # SETUP CREDENTIALS
+    credentials = Credentials(
+        int_account=None,
+        username=config.DG_USERNAME,
+        password=config.DG_PASSWORD,
+        one_time_password=config.DG_TOTP,
+        totp_secret_key=config.DG_TOTP_SECRET,
+    )
+
+    # SETUP CONTROLLER
+    degiro_controller = DegiroController(credentials=credentials)
+    degiro_controller.help()
+
+    while True:
+        # Get input command from user
+        if session and gtff.USE_PROMPT_TOOLKIT:
+            completer = NestedCompleter.from_nested_dict(
+                {c: None for c in degiro_controller.CHOICES}
+            )
+            an_input = session.prompt(
+                f"{get_flair()} (pa) > (degiro)> ",
+                completer=completer,
+            )
+        else:
+            an_input = input(f"{get_flair()} (pa) > (degiro)> ")
```

Comment:
Should be:
"(pa)>(degiro)> "

instead

Reply:
Changed